README.md

```diff
@@ -39,6 +38,51 @@ query = gql('''
 client.execute(query)
 ```
 
+To execute against a graphQL API. (We get the schema by using introspection).
+
+```
```

Comment:
Adding `python` after these backticks would make this easier to read. Same for line 67 below.